arbiter-core/src/middleware.rs

```diff
@@ -549,10 +604,7 @@ impl JsonRpcClient for Connection {
                             logs.push(log);
                         }
                     }
-                    break;
                 }
-
-                // TODO: This can probably be avoided somehow
```

Comment:
Okay a nitpick but I think this TODO is valid. Why did you delete it?

Reply:
I agree i think it is still valid. Do you think we can we make an issue for it instead of leaving it in so that the code base is a little cleaner?

Reply:
I don't mind a `TODO` here or there in a codebase. But we should also make an issue for it, yeah.